src/Components/DialPad.js

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
-import './index.css';
+import '../index.css';
 
-function Display() {
+function DialPad() {
   return (
-    <div className="container">
-      <h1>Calculator</h1>
-      <h2>0</h2>
+    <div>
       <div className="board-row">
         <button type="button">AC</button>
         <button type="button">+/-</button>
```

Comment:
- [ ] The ButtonPanel (`DialPad`) component should render the calculator panel using the Button (`SingleButton`) component, not the html `<button/>` tag.
Screenshot of requirements 👇🏾 
<img width="404" alt="Screen Shot 2021-03-23 at 12 52 48 AM" src="https://user-images.githubusercontent.com/48804258/112072434-1a608d80-8b72-11eb-9838-924afd3e9860.png">